README.md

```diff
@@ -1,2 +1,37 @@
-# AR6-WG1-Data-Compilation
-Compilation of data from AR6 WG1
+# Data Compilation & Figures for the IPCC AR6 WG1
+
+© 2021 Zebedee Nicholls & Daniel Huppmann; licensed under the [BSD 3-Clause License](LICENSE).
+
+[![license](https://img.shields.io/github/license/openscm/ar6-wg1-data-compilation)](https://github.com/IAMconsortium/pyam/blob/main/LICENSE)
+
+## Overview
+
+The IPCC's [AR6 WG1 Report](https://www.ipcc.ch/report/ar6/wg1/)
+published in August 2021 contains a lot of useful data and figures.
+The data is available at the [CEDA Archive](https://data.ceda.ac.uk/badc/ar6_wg1/data/)
+under a Creative Commons CC-BY license, but the formats are quite diverse
+and not easy to handle.
+
+### Data resources
+
+This repository compiles the data into a uniform, csv-based data format
+following the standard established by the Integrated Assessment Modeling Consortium
+([IAMC](https://www.iamconsortium.org)) and used by IPCC WG3.
+
+The format used in this repository is directly compatible with
+the **scmdata** and **pyam** Python packages (see dependencies below),
+but can be easily read with Excel or scripts written on other programming languages.
```

Comment:
```suggestion
but can be easily read with Excel or scripts written in other programming languages.
```

Reply:
Maybe add something like:

The processed data is stored in `data/processed` (see also `data/README.md`). For complete transparency and reproducibility, all steps to compile the processed data from the raw data are also included in this repository (not true yet, but will be true once we start adding stuff). We will add instructions about how to compile the processed data from the raw data in future pull requests.

Reply:
Sounds good, some inspo text for you then